fixtures/v2/api/getDomainRegistration/success.http

```diff
@@ -0,0 +1,20 @@
+HTTP/1.1 200 OK
+Server: nginx
+Date: Fri, 05 Jun 2020 18:23:53 GMT
+Content-Type: application/json; charset=utf-8
+Connection: keep-alive
+X-RateLimit-Limit: 2400
+X-RateLimit-Remaining: 2392
+X-RateLimit-Reset: 1591384034
+ETag: W/"80c5827934c13b1ca87a587d96e7d1e8"
+Cache-Control: max-age=0, private, must-revalidate
+X-Request-Id: 9f4959ee-06a9-488c-906e-27a570cafbbf
+X-Runtime: 0.078429
+X-Frame-Options: DENY
+X-Content-Type-Options: nosniff
+X-XSS-Protection: 1; mode=block
+X-Download-Options: noopen
+X-Permitted-Cross-Domain-Policies: none
+Strict-Transport-Security: max-age=31536000
+
+{"data":{"id":361,"domain_id":104040,"registrant_id":2715,"period":1,"state":"registering","auto_renew":false,"whois_privacy":false,"estimated_completion_at":"2023-02-01T17:44:32Z","created_at":"2023-01-27T17:44:32Z","updated_at":"2023-01-27T17:44:40Z"}}

```

Comment:
The `estimated_completion_at` attribute would need to be added to the component schemas of the `DomainRegistration`, `DomainRenewal`, and `DomainTransfer`.